apiserver/facades/client/client/instanceconfig.go

```diff
@@ -63,10 +63,10 @@ func InstanceConfig(st *state.State, machineId, nonce, dataDir string) (*instanc
 		Arch:         *hc.Arch,
 	})
 	if err != nil {
-		return nil, errors.Annotate(err, "finding tools")
+		return nil, errors.Annotate(err, "finding agents")
```

Comment:
generally, reading through these, the word "agent" feels like it should be treated as an adjective rather than a noun, so if we want a plural it would be "agent binaries".
We *can* do "agents" but that generally refers to the running 'jujud' that is operating on behalf of a machine or unit, vs the tgz file that holds the binary that we want to run. Thoughts?